packages/material-ui/src/Select/selectClasses.js

```diff
@@ -0,0 +1,21 @@
+import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unstyled';
+
+export function getSelectUtilitiyClasses(slot) {
```

Comment:
Nitpick: typo. 🙂 `getSelectUtilityClasses`

Reply:
fixed in #25804